packages/notification-service/package.json

```diff
@@ -37,8 +38,11 @@
     "@types/dotenv": "^4.0.3",
     "@types/express": "^4.16.1",
     "@types/i18next": "^11.9.3",
+    "@types/jest": "^24.0.16",
     "@types/node-fetch": "^2.1.2",
     "@types/utf8": "^2.1.6",
-    "@types/web3": "^1.0.18"
+    "@types/web3": "^1.0.18",
+    "jest": "^24.8.0",
+    "ts-jest": "^24.0.2"
```

Comment:
Please use the same versions as the other projects (see mobile package.json). Difference dep versions across our packages can cause unexpected issues